packages/semantic/src/release.config.ts

```diff
@@ -107,7 +25,9 @@ const config = {
     [
       '@semantic-release/commit-analyzer',
       {
-        parserOpts,
+        parserOpts: {
+          noteKeywords: parserOpts.noteKeywords,
```

Comment:
Big lol. Was passing too many props and it was 🥁 _breaking_ 🥁  the breaking change check on types.